core/test/audits/prioritize-lcp-image-test.js

```diff
@@ -52,7 +59,9 @@ describe('Performance: prioritize-lcp-image audit', () => {
         networkRequestTime: 0,
         networkEndTime: 500,
         timing: {receiveHeadersEnd: 500},
+        transferSize: 400,
```

Comment:
these all really needed transfer sizes because otherwise they all completed in a single round trip making any dependency changes difficult to differentiate